hw/ip_templates/rv_plic/fpv/tb/coverage_waivers.tcl.tpl

```diff
@@ -41,3 +41,12 @@ check_cov -waiver -add -source_file {../src/lowrisc_prim_subreg_0/rtl/prim_subre
 check_cov -waiver -add -source_file {../src/lowrisc_prim_subreg_0/rtl/prim_subreg.sv} -start_line\
  58 -end_line 58 -type {branch} -comment {wr_en is true and the branch doesn't contain the else\
  part}
+
+# Not possible to have out of list state without FI.
```

Comment:
I know this is a new question, but would it make sense to try using `stopat` to make this possible? (I *think* the result will consider the code covered)

Reply:
I am curious, there is no mechanism in FPV to inject faults? For our chip there are many side-effects of error injection, like alerts issued and so on. We check them in simulation, but it would be nice if FPV was also able to check them.

Reply:
That mechanism *is* `stopat`!